src/domains/job/search/components/form/WorkExperience.jsx

```
@@ -11,7 +11,7 @@ const WorkExperience = ({ register, errors }) => {
 
       <div className='flex items-start'>
         {errors.experience && (
```

ESLint

[error] 'errors.experience' is missing in props validation

(react/prop-types)